main/build.gradle

```diff
@@ -85,11 +110,17 @@ dependencies {
     implementation 'com.google.flogger:flogger-system-backend:0.6'
     implementation 'com.univocity:univocity-parsers:2.9.0'
     implementation 'com.google.geometry:s2-geometry:2.0.0'
+    implementation 'com.sparkjava:spark-core:2.9.3' // Spark web framework.
```

Comment:
This dependency for Spark can be removed?

Reply:
Yes absolutely